lib/models/repository.js

```diff
@@ -247,6 +247,24 @@ export default class Repository {
       : nullAuthor;
   }
 
+  // todo (@annthurium, 3/2019): refactor GitHubTabController etc to use this method.
```

Comment:
Found another todo. Currently `GitHubTabController` is _not_ using this method... only the `CommentDecorationsContainer`

Reply:
Filed as #1047.